doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,305 @@
+      "properties": {
+        "language": {
+          "type": "string",
+          "description": "The programming language used to write this connector"
+        },
+        "languageVersion": {
+          "type": "string",
+          "description": "The version of the programming language used to write this connector"
+        },
+        "sdk": {
+          "type": "object",
+          "description": "Details about the versions of the AIO SDKs used to write this connector (if they were used)",
+          "properties": {
+            "mqttPackageVersion": {
+              "type": "string",
+              "description": "The version of the azure iot operations MQTT package"
+            },
+            "servicesPackageVersion": {
+              "type": "string",
+              "description": "The version of the azure iot operations services package"
+            },
+            "connectorPackageVersion": {
+              "type": "string",
+              "description": "The version of the azure iot operations connector package"
+            }
+          }
+        }
+      },
+      "required": [
+        "language",
+        "languageVersion"
+      ]
+    },
+    "resourceRequirements": {
+      "type": "object",
+      "description": "The minimal resource requirements to run this connector",
+      "properties": {
+        "cpu": {
+          "type": "string"
+        },
+        "memory": {
+          "type": "string"
+        },
+        "storage": {
+          "type": "string"
+        }
+      }
+    },
+    "aioMetadata": {
+      "type": "object",
+      "description": "The AIO requirements to run this connector",
+      "properties": {
+        "aioMinVersion": {
+          "type": "string",
+          "description": "The minimal (inclusive) version that can run this connector."
+        },
+        "aioMaxVersion": {
+          "type": "string",
+          "description": "The maximal (inclusive) version that can run this connector."
+        }
+      }
+    },
+    "inboundEndpoints": {
+      "type": "array",
+      "description": "The array of inbound endpoints that this connector connects with.",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "configSchemas": {
+              "type": "object",
+              "description": "The inbound endpoint level JSON schemas for the various 'additional configuration' fields. A JSON schema must be provided for each 'additional configuration' only if that field is used by this connector.",
+              "properties": {
+                "deviceEndpointAdditionalConfigSchema": {
+                  "$ref": "#/definitions/additionalConfigurationType"
+                }
+              }
+            },
+            "assets": {
+              "type": "array",
+              "description": "The array of assets within an inbound endpoint",
+              "items": [
```

Comment:
what do you mean with this? there are no assets within an inbound endpoint.

Reply:
I re-worded this a bit